exercises/linked-list/.meta/src/reference/java/DoublyLinkedList.java

```diff
@@ -43,12 +43,12 @@ public T shift() {
         return value;
     }
 
-    private static final class Element<T> {
```

Comment:
I think that this nested class can probably stay as `private`, unless it causes any problems! :slightly_smiling_face:

Reply:
I understand. I have committed the requested changes